presto-orc/src/main/java/com/facebook/presto/orc/reader/SliceDictionarySelectiveReader.java

```diff
@@ -602,4 +635,11 @@ private BlockLease newLease(Block block)
         valuesInUse = true;
         return ClosingBlockLease.newLease(block, () -> valuesInUse = false);
     }
+
+    private class Status
```

Comment:
Why not use Enum here?

Reply:
I had a enum before but Masha suggested to use a byte fields similar to FilterFunctions

Reply:
Why not making those constants public so we can remove this class?

Reply:
These are status specific constants, isn't it better to keep them in inner class so that its more clear on what these are, are there any performance implications, by keeping them in a private static inner class. lmk

Reply:
If these statuses have nothing to do with `FilterFunction`, can we put them as constants for `SliceDictionarySelectiveReader` rather than an inner class? After all the inner class is not used but only the bytes.

Reply:
Could you add a comment explaining why we're using a byte here? For example:
`The return status is represented as a byte to achieve better performance, because the method returning this value is in performance critical loops and using Enum would incur high cost of constructing a class object.`

Reply:
using byte constants instead of enum as its memory efficient